src/assets/stylesheets/2d-hud.scss

```diff
@@ -393,3 +367,204 @@
     background-image: url(../hud/share_window_on.png);
   }
 }
+
+// Discrete frames of the mic "animation"
+:local(.mic-0) {
```

Comment:
Can remove these now that the css spritesheet is working without the "bouncing icon" issue